search-ui/tests/unit/Breadcrumb.spec.ts

```diff
@@ -0,0 +1,50 @@
+// Libraries
+import { RouteNames } from '@/enums'
+import { mount } from '@vue/test-utils'
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
+import { BcrsBreadcrumb } from '@/bcrs-common-components'
+import { SearchDashboardBreadcrumb, SearchSIGNINBreadcrumb } from '@/bcrs-common-components/resources'
+import store from '@/store'
+
+
+describe('Breadcrumb.vue', () => {
+
+  let wrapper;
+  const breadcrumbs : BreadcrumbIF[] = [
+    {
+      text: SearchSIGNINBreadcrumb.text,
+      to: { name: RouteNames.SIGN_IN }
+    },
+    {
+      text: SearchDashboardBreadcrumb.text,
+      to: { name: RouteNames.DASHBOARD }
+    }
+  ]
+
+  wrapper = mount(BcrsBreadcrumb, {
+    props: { breadcrumbs },
+    global: {
+      provide: {
+        store: store
+      },
+      mocks: {
+        $route: {
+          name: RouteNames.DASHBOARD
+        }
+      }
+    },
+    shallow: true  // stubs out children components
+  })
+
+  it('present page is dashboard so should display breadcrumb', () => {
+    expect(wrapper.exists()).toBe(true)
+  })
+
+  it('dashboard back should be redirect to sign in', () => {
```

Comment:
dashboard back should redirect you to the REGISTRY_URL config value

Reply:
Since you can't simulate the click on the button due to vuetify limitations, can you simulate going to the value from the function (as if someone had clicked it in the UI) and checking the location goes as expected

Reply:
I am now able to test the URL in test and also on UI as well. Please check!